app/controllers/teaspoon/bundle/bundle_controller.rb

```diff
@@ -4,7 +4,7 @@ def serve
       file = File.join(Teaspoon::Bundle.dir, "teaspoon_bundle_#{params[:suite]}.js.erb")
       File.write(file, contents)
       begin
-        asset = Rails.application.assets.find_asset(file)
+        asset = Rails.application.assets.find_asset(file, accept: 'application/javascript')
```

Comment:
Sprockets 4 changed from registering the erb compiler as a processor to registering it as a transformer. Transformers are only run when the extension type differs from the accepted type. If accept isn't specified it defaults to the same as the extension (`application/javascript+ruby`) and the transformer isn't run, thus serving uncompiled `js.erb`.